content/1.getting-started/3.contribution.md

```diff
@@ -106,6 +106,36 @@ Your Vue components should adhere to the following structure:
 </style>
 ```
 
+**Props typing and code style**
+
+Refer to this Vue.js documentation page -> https://vuejs.org/api/sfc-script-setup#type-only-props-emit-declarations
+
+```vue
+<script lang="ts" setup>
+// DON'T ⛔️
+const props = defineProps({
+  whatever: { type: String, required: true },
+  optional: { type: String, default: "default" },
+});
+
+// DO ✅
+interface Props {
```

Comment:
Hi @nathan-de-pachtere,
Just a question, we are preferring using types over interface, what's your thought on this?

Reply:
I'm used to interfaces and classes because I code in strongly-typed languages like PHP and C#. Interfaces make more sense to me, and they feel more modern. But at the end of the day, I don't mind as long as the codebase is consistent inside a project scope. ;)

https://www.youtube.com/watch?v=zM9UPcIyyhQ&ab_channel=MattPocock

Reply:
Cool. Then I suggest we move with using `types` as primary standard and `interface` as secondary. Can you please update the example documentation?

Reply:
Feels wrong to have an "=" operator for an "object contract" as it's not a variable 🫣 Anyway it's in the nature of javascript/typescript to be weird, I'm going to be used to it 🥲 Changes done